common/webapp/src/commonMain/kotlin/models/HandlingResult.kt

```diff
@@ -9,10 +9,29 @@ sealed interface HandlingResult<R : Any?> {
     val code: HttpStatusCode
     val data: R
     @Serializable
-    data class Success<R : Any?>(override val data: R, override val code: @Serializable(HttpStatusCodeSerializer::class) HttpStatusCode = HttpStatusCode.OK) : HandlingResult<R>
+    data class Success<R : Any?>(
+        override val code: @Serializable(HttpStatusCodeSerializer::class) HttpStatusCode = HttpStatusCode.OK,
+        override val data: R
```

Comment:
In `Success` data is more important than status code, so, it must be the first one